--- origsrc/llpp/main.ml	2012-08-21 01:42:22.000000000 -0500
+++ src/llpp/main.ml	2012-08-21 02:33:49.626533000 -0500
@@ -509,18 +509,16 @@ let defconf =
   ; aalevel        = 8
   ; urilauncher    =
       (match platform with
-      | Plinux | Pfreebsd | Pdragonflybsd
+      | Plinux | Pfreebsd | Pdragonflybsd | Pcygwin
       | Popenbsd | Pnetbsd | Psun -> "xdg-open \"%s\""
       | Posx -> "open \"%s\""
-      | Pcygwin -> "cygstart \"%s\""
       | Punknown -> "echo %s")
   ; pathlauncher   = "lp \"%s\""
   ; selcmd         =
       (match platform with
-      | Plinux | Pfreebsd | Pdragonflybsd
+      | Plinux | Pfreebsd | Pdragonflybsd | Pcygwin
       | Popenbsd | Pnetbsd | Psun -> "xsel -i"
       | Posx -> "pbcopy"
-      | Pcygwin -> "wsel"
       | Punknown -> "cat")
   ; colorspace     = Rgb
   ; invert         = false
