--- origsrc/llpp/config.ml	2015-11-29 01:55:20.000000000 -0600
+++ src/llpp/config.ml	2015-11-29 01:57:36.284666800 -0600
@@ -521,16 +521,14 @@ let defconf =
   ; aalevel        = 8
   ; urilauncher    =
       (match platform with
-      | Plinux | Psun | Pbsd -> "xdg-open \"%s\""
+      | Plinux | Psun | Pbsd | Pcygwin -> "xdg-open \"%s\""
       | Posx -> "open \"%s\""
-      | Pcygwin -> "cygstart \"%s\""
       | Punknown -> "echo %s")
   ; pathlauncher   = "lp \"%s\""
   ; selcmd         =
       (match platform with
-      | Plinux | Pbsd | Psun -> "xsel -i"
+      | Plinux | Pbsd | Psun | Pcygwin -> "xsel -i"
       | Posx -> "pbcopy"
-      | Pcygwin -> "wsel"
       | Punknown -> "cat")
   ; paxcmd         = "cat"
   ; passcmd        = E.s
